### DISCUSSION TITLE
Making new NavigationGroup

### DISCUSSION THREAD

**Discussion by waltermiglioress** [filamentphp/filament#7645] [Help]
Hello everyone,
this is the first time I write here.First of all sorry for my english  mistakes. I hope someone can help me. Is there a way to make something like a new "menu" or Navigation Group and register it to the topbar section (through the render hook I can imagine)?
I would like to implement either both top navigation and the sidebar nav putting some policy (with spatie permission package) showing conditional sidebar menu items depending on the main topbar item.
Thanks for the tips!

**Comment by pboivin** [comment]
Hi @waltermiglioress, I don't think it's currently possible to use both the sidebar and the top nav but, if you keep the sidebar as your main navigation, you can inject a custom menu or dropdown in the header with a render hook like `panels::global-search.start`:
https://filamentphp.com/docs/3.x/support/render-hooks
